src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderTransactionHistoryResourceTest.java

```diff
@@ -232,8 +232,8 @@ public void When_GetOffenderTransactionHistory_And_BadFromDateFormat_Then_ErrorR
             OFFENDER_NO);
 
         assertThat(response.getBody().getStatus().intValue()).isEqualTo(HTTP_BAD_REQ);
-        assertThat(response.getBody().getDeveloperMessage()).isEqualTo("Invalid value for MonthOfYear (valid values 1 - 12): 30");
```

Comment:
had to change test here due to spring boot upgrade

Reply:
note that the response body will change for clients, but very dubious that any will notice since still returns the bad request response as expected

Reply:
@petergphillips do we not get a body anymore or is there some other problem?

Reply:
we do, it is just that the validation message is now different.